### ISSUE TITLE
enhance `RPC` to support primitive types and `null`

### ISSUE THREAD

**Issue by libenchao** [TenFifteen/TDFS#6]
RT

**Comment by libenchao** [comment]
@yanyiping  
I have noticed that, our `RPC` can deal with primitive types and `null`. So there is no need to do this task.
My bad.

PS: you can explain to us why it can work with primitive types and `null`.

cc @songpengwei

**Comment by yanyiping** [comment]
@libenchao @songpengwei 
Bravo~That means Chao did a good job with easy code. 
I'll explain it later.

**Comment by songpengwei** [comment]
But you can implement some mechanism to support more customized Class. hahaha @yanyiping

**Comment by yanyiping** [comment]
Sounds hard for me. Anyway, I'll try. @songpengwei

**Comment by libenchao** [comment]
what's the meaning of 'more customized Class'? @songpengwei

**Comment by songpengwei** [comment]
Maybe I mistaken it, can `Param` and `writeObject` support the class defined by ourselves? @libenchao

**Comment by libenchao** [comment]
Of course, any class implements `Serializable` can be used in our `RPC`. @songpengwei
